docs/networking/acns-lab.md

```diff
@@ -198,79 +288,811 @@ Let's start by applying a new network policy to cause some chaos in the network.
+4. Visualize traffic patterns over time to pinpoint when the issue started
+5. **Estimated time: 10-15 minutes** to fully diagnose the root cause
+
+Let's see this in action by investigating the issues developers reported.
+
+#### Enable Flow Logs for the Pets Namespace
+
+To enable container network flow logs, you need to apply a `ContainerNetworkLog` custom resource that defines which network flows to capture. Let's create a filter to capture all traffic in the pets namespace.
+
+Create a file named `pets-flow-logs.yaml` with the following content:
+
+```yaml
+apiVersion: acn.azure.com/v1alpha1
+kind: ContainerNetworkLog
+metadata:
+  name: testcnl # Cluster scoped
+spec:
+  includefilters: # List of filters
+    - name: egress-filter # Capture egress traffic from pets namespace
+      from:
+        namespacedPod: # List of source namespace/pods. Prepend namespace with /
+```bash
+apiVersion: acn.azure.com/v1alpha1
+kind: ContainerNetworkLog
+metadata:
+  name: testcnl # Cluster scoped
+spec:
+  includefilters: # List of filters
+    - name: egress-filter # Capture egress traffic from pets namespace
+      from:
+        namespacedPod: # List of source namespace/pods. Prepend namespace with /
+          - pets/order-service-
+          - pets/product-service-
+          - pets/rabbitmq
+          - pets/store-front-
+      protocol: # List of protocols; can be tcp, udp, dns
+        - tcp
+        - udp
+        - dns
+      verdict: # List of verdicts; can be forwarded, dropped
+        - forwarded
+        - dropped
+    
+    - name: ingress-filter # Capture ingress traffic to pets namespace
+      to:
+        namespacedPod: # Destination pods
+          - pets/store-front-
+          - pets/order-service-
+          - pets/product-service-
+          - pets/rabbitmq
+      protocol:
+        - tcp
+        - udp
+      verdict:
+        - forwarded
+        - dropped
+```
+
+Apply the custom resource to enable flow log collection:
+
+```bash
+kubectl apply -f pets-flow-logs.yaml
+```
+
+Verify the custom resource was created successfully:
 
-[Optional] Familiarize yourself with the other dashboards for DNS, and pod flows
+```bash
+kubectl describe containernetworklog testcnl
+```
+
+You should see a `Status` field showing `State: CONFIGURED`. This means flow logs are now being collected for the pets namespace and sent to your Log Analytics workspace.
+
+<div class="info" data-title="Note">
+
+> Flow logs are stored locally on the nodes at `/var/log/acns/hubble/events.log` and then collected by the Azure Monitor Agent and sent to Log Analytics. It may take 2-3 minutes for logs to appear in Log Analytics after network events occur.
+
+</div>
+
+#### Generate Traffic to Observe Flow Logs
+
+> **Note:** Flow logs are stored locally on the nodes at `/var/log/acns/hubble/events.log` and then collected by the Azure Monitor Agent and sent to Log Analytics. It may take 2-3 minutes for logs to appear in Log Analytics after network events occur.
```

Comment:
Docusaurus admonition

```suggestion
:::info

Flow logs are stored locally on the nodes at `/var/log/acns/hubble/events.log` and then collected by the Azure Monitor Agent and sent to Log Analytics. It may take 2-3 minutes for logs to appear in Log Analytics after network events occur.

:::
```